Blockchain.info Typo Fixes

todofixthis opened the pull request:
- Fixed capitalization. (x3)
- Made references to Blockchain.info consistent.
- Re-worded sentence for clarity. (x4)
- Fixed invalid `<img>` tag syntax.*
- Changed references from "us" to "you" (e.g., "your" instead of "our", etc.) to be consistent with the rest of the site. (x3)
- Fixed inaccurate content (s/b "claim your BTS", not "claim your AGS").
- Added content that is relevant to Mac OS X users.
- For multi-sentence paragraphs, put each sentence on its own line to make merging changes easier (does not affect display). (x2)*
- Converted all path-relative URLs in img src attributes to absolute so that the page can be moved anywhere in the sitemap.*

\* "meta" fixes that do not affect how the content is displayed in browsers.

PTS addy PZ32Dv8NA1CBNtYyu5Z14qWNL6VWDSP4HC

bytemaster commented:
Great list of changes... this repo is no longer the official invictus repo for our website, can you resubmit the pull request to github.com/InvictusInnovations/invictus.io

Commit:
Misc. fixes on bitshares-ags page.

- Absorbed sentence fragment into introductory sentence.
- Fixed run-on sentence.
- Made link to blockchain.info page more resilient to sitemap reorgs*

\* Indicates "meta" fix that does not affect how the content is displayed.

Commit:
Merge branch 'master' of https://github.com/InvictusInnovations/invictus.io
